### DISCUSSION TITLE
Score Features page has been added!

### DISCUSSION THREAD

**Discussion by conorwild** [TheOwenLab/cbs-toolkit#3] [General]
Check it out: [Score Features](https://github.com/TheOwenLab/cbs-toolkit/wiki/Score-Features)

It's not pretty, but all the information is there! Also, I need to review it to make sure it's up-to-date. This should be super useful for everyone. 

Any thoughts on how to make it better? Should we separate the tasks into different pages, or have clearer sections?

cc @mike-battista

**Comment by mike-battista** [comment]
@conorwild This is fantastic! Thank you! 

Is there a chance something like this existing for the SART task? More and more researchers are using that task, and it may make its way to CBS Health as well.
